### DISCUSSION TITLE
Proxmox VE LXC Dist-Upgrade

### DISCUSSION THREAD

**Discussion by vdias** [tteck/Proxmox#1534] [Ideas]
Could be really great to have a script to upgrade all LXC system to a newer version...

Just upgraded my proxmox enviroment to Debian Bookworm, but now i should upgrade all my LXC to bookworm too... and we need to go one be one changing the sources and doing the dist-upgrade...

**Comment by tteck** [comment]
>Updating the LXC to Debian 12 manually could potentially cause a loss of functionality.

https://github.com/tteck/Proxmox/discussions/1498
